ui.py

```diff
@@ -62,6 +63,7 @@ def apply_settings(self):
 
         self._send_to_device = prefs['send_to_device']
         self._create_xray_when_sending = prefs['create_xray_when_sending']
+        self._expand_aliases = prefs['expand_aliases']
```

Comment:
If we need to save all the config in two places (ie. "init" (ok, genesis) and "apply_settings") then having a worker function seems worthwhile to avoid C&P bugs.